CommonMark/Formatters/HtmlFormatterSlim.cs

```diff
@@ -621,6 +628,15 @@ private static void InlinesToHtml(HtmlTextWriter writer, Inline inline, CommonMa
                         stackWithinLink = withinLink;
                         break;
 
+                    case InlineTag.Placeholder:
+                        // the slim formatter will treat placeholders like literals, without applying any further processing
```

Comment:
To avoid this, couldn't the resolver be given in the settings instead of the HtmlFormatter class so that the slim version could make use of it as well?